doc/src/pair_fep_soft.txt

```diff
@@ -228,40 +294,57 @@ instructions on how to use the accelerated styles effectively.
+The {morse/soft} pair style does not support mixing. Thus, coefficients for all
+LJ pairs must be specified explicitly.
 
-These pair styles write information to "binary restart
-files"_restart.html, so pair_style and pair_coeff commands do not need
-to be specified in an input script that reads a restart file.
+All of the pair styles with soft core support the "pair_modify"_pair_modify.html
+shift option for the energy of the Lennard-Jones portion of the pair
+interaction.
 
-:line
+The different versions of the {lj/cut/soft} pair styles support the
+"pair_modify"_pair_modify.html tail option for adding a long-range tail
+correction to the energy and pressure for the Lennard-Jones portion of the pair
+interaction.
 
-[Restrictions:]
+NOTE: The analytical form of the tail corrections for energy and pressure used
+in the {lj/cut/soft} potentials are approximate, being identical to that of the
+corresponding non-soft potentials scaled by a factor lambda^n. The errors due to
+this approximation should be negligible. For example, for a cutoff of 2.5 sigma
+this approximation leads to maximum relative errors in tail corrections of the
+order of 1e-4 for energy and virial (alpha_LJ = 0.5, n = 2). The error vanishes
+when lambda approaches 0 or 1. Note that these are the errors affecting the
+long-range tail (itself a correction to the interaction energy) which includes
+other approximations, namely that the system is homogeneous (local density equal
+the average density) beyond the cutoff.
+
+The {morse/soft} pair style does not support the "pair_modify"_pair_modify.html
+tail option for adding long-range tail corrections to energy and pressure.
 
-To avoid division by zero do not set sigma = 0; use the lambda
-parameter instead to activate/deactivate interactions, or use
-epsilon = 0 and sigma = 1. Alternatively, when sites do not
-interact though the Lennard-Jones term the {coul/long/soft} or
-similar sub-style can be used via the
-"pair_style hybrid/overlay"_pair_hybrid.html command.
+All of these pair styles write information to "binary restart
+files"_restart.html, so pair_style and pair_coeff commands do not need to be
+specified in an input script that reads a restart file.
 
 :line
 
-All of the plain {soft} pair styles are part of the USER-FEP package.
-The {long} styles also requires the KSPACE package to be installed.
-They are only enabled if LAMMPS was built with those packages.  See
-the "Build package"_Build_package.html doc page for more info.
+[Restrictions:]
+
+The pair styles with soft core are only enabled if LAMMPS was built with the
+USER-FEP package. The {long} versions also require the KSPACE package to be
+installed. The soft {tip4p} versions also require the MOLECULE package to be
+installed. These styles are only enabled if LAMMPS was built with those
+packages.  See the "Build package"_Build_package.html doc page for more
+info. Note that the KSPACE and MOLECULE packages are installed by default.
```

Comment:
>Note that the KSPACE and MOLECULE packages are installed by default.

This isn't true if you are using the legacy build system and building from source, right? Then no packages are installed.

Reply:
Ok. But the same is written in pair lj/cut/coul/long (the canonical pair style for me)
```suggestion
- info. Note that the KSPACE and MOLECULE packages are installed by default.
+ info.
```

Reply:
Ok. But the same is written in pair lj/cut/coul/long (sort of a canonical pair style for me...)
```suggestion
info.
```

Reply:
both need to be changed. since the addition of CMake support for building LAMMPS, the tar archives have no more packages pre-installed, as that would collide with how CMake works.
i will take care of it and try to hunt down other instances of this as well.

Reply:
fixed in 14ab0ec4593dc91e4455d2e05c74ab20b0e64225